File: Programmers/pro_42586_기능개발.py
def solution(progresses, speeds):
    answer = [0]
    day=0
    for i,j in zip(progresses,speeds):
        if(i+j*day>=100):            
            answer[-1]+=1
        else:
            day = (100-i)//j if (100-i)%j ==0 else ((100-i)//j)+1
            answer.append(1)
    return answer if answer[0]!=0 else answer[1:]

File: Programmers/pro_42586_기능개발.js
function solution(progresses, speeds) {
    var answer = [0];
    const len = progresses.length
    let day = 0
    for (let i = 0; i < len; i++) {
        if (progresses[i] + day * speeds[i] >= 100)
            answer[answer.length - 1] += 1
        else {
            day = Math.ceil((100 - progresses[i]) / speeds[i])
            answer.push(1)
        }
    }
    return answer[0] === 0 ? answer.slice(1) : answer;
}